tests/providers/test_ssn.py

```diff
@@ -876,6 +876,33 @@ def test_pt_BR_rg(self):
                 assert re.search(r"^\d{9}$", to_test)
 
 
+class TestNlBE(unittest.TestCase):
```

Comment:
moved from test_factory.py

Reply:
Thank you for cleaning this up, but could you make these changes in a separate PR?

Reply:
No problem.

Reply:
See #1606.

Reply:
I appreciate it. #1606 is merged :)